Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -693,15 +693,15 @@ MotionVector Decoder::select_motion_vector(u8 plane, BlockContext const& block_c
     auto round_mv_comp_q2 = [&](MotionVector in) {
         // return (value < 0 ? value - 1 : value + 1) / 2
         return MotionVector {
-            (in.row() < 0 ? in.row() - 1 : in.row() + 1) >> 1,
-            (in.column() < 0 ? in.column() - 1 : in.column() + 1) >> 1
+            (in.row() < 0 ? in.row() - 1 : in.row() + 1) / 2,
```

Comment:
Cool, I think this will do the right thing now (https://godbolt.org/z/TcTM7Eszq) , and it's nice and readable and looks like the spec. The commit description now needs updating though :)

Reply:
Fixed!